src/generation.rs

```diff
@@ -61,6 +61,16 @@ impl Template {
         Template { path: buf }
     }
 
+    fn render_template(&self, content: &str, context: &Context, path: Option<PathBuf>) -> Result<String> {
+        let mut tera = Tera::default();
+
+        let content = content.replace("$$", "|");
```

Comment:
you only want to do that for filenames/directories, is that function used for other things?

Reply:
My original goal was to enable the filtering capability in paths, but supporting this alternative syntax everywhere would be better to avoid ambiguity around where the `$$` may be used, I'd say.

Reply:
You might change legit content in a file by doing that. This feature is only there to allow calling filters on files, no need to expand it beyond that.

Reply:
That's a fair point, give me 2 minutes to move that.

Reply:
(I'll also update the readme to clear the ambiguity)